code/index.html

```diff
@@ -2,17 +2,137 @@
+         <div class="middle">
+        <h2 class="text">Champagne Cake Pops</h2>
+        </div></a>
+      </div>
+
+      <div class="flex-item-home">
+        <a href="r_comingsoon.html"><img src="./images/baking/tobleronemousse.jpg" alt="Toblerone Mousse" class="image">
+         <div class="middle">
+        <h2 class="text">Toblerone Mousse</h2>
+        </div></a>
+      </div>
+
+      <div class="flex-item-home">
+        <a href="r_comingsoon.html"><img src="./images/baking/creamcheesemuffin.jpg" alt="Cream Cheese Muffins" class="image">
+         <div class="middle">
+        <h2 class="text">Cream Cheese Muffins</h2>
+        </div></a>
+      </div>
+
+
+      <div class="flex-item-home">
+        <a href="r_comingsoon.html"><img src="./images/baking/brownies2.jpg" alt="Zucchini Brownies" class="image">
+         <div class="middle">
+        <h2 class="text">Zucchini Brownies</h2>
+        </div></a>
+      </div>
+
+    
+      <div class="flex-item-home">
+        <a href="r_comingsoon.html"><img src="./images/baking/tiramisu.jpg" alt="Tiramisu" class="image">
+         <div class="middle">
+        <h2 class="text">Tiramisu</h2>
+        </div></a>
+      </div>
+
+      <div class="flex-item-home">
+        <a href="r_comingsoon.html"><img src="./images/baking/chocolatechipcookies.jpg" alt="Chocolate Chip Cookies" class="image">
+         <div class="middle">
+        <h2 class="text">Chocolate Chip Cookies</h2>
+        </div></a>
+      </div>
+
+
+      <div class="flex-item-home">
+        <a href="r_comingsoon.html"><img src="./images/baking/cremebrulee.jpg" alt="Crème Brûlée" class="image">
+         <div class="middle">
+        <h2 class="text">Crème Brûlée</h2>
+        </div></a>
+      </div>
+    </div>
+
+
+    <footer>
+    <div>
+    <a href="https://web.whatsapp.com" target="_blank"><img src="./images/icon-social/social-icon-whatsapp.jpg" alt="Whatsapp" class="social-icon"></a>
+    <a href="https://twitter.com" target="_blank"><img src="./images/icon-social/social-icon-twitter.jpg" alt="Twitter" class="social-icon"></a>
+    <a href="https://facebook.com" target="_blank"><img src="./images/icon-social/social-icon-fb.jpg" alt="Facebook" class="social-icon"></a>
+    <a href="https://instagram.com" target="_blank"><img src="./images/icon-social/social-icon-insta.jpg" alt="Instagram" class="social-icon"></a>
+    <a href="https://pinterest.com" target="_blank"><img src="./images/icon-social/social-icon-pinterest.jpg" alt="Pinterest" class="social-icon"></a>
+    <a href="https://youtube.com" target="_blank"><img src="./images/icon-social/social-icon-youtube.jpg" alt="Youtube" class="social-icon"></a>
+    <h2 class="footer-h2">Stressed is just desserts spelled backwards</h2>
+    </div>
+ <hr>
+ <div>
+ <p class="p-footer">Page created 10/2020 by Mirjam B. for <a href="https://codeexcursion.ch" target="_blank">Code Excursion</a></p>
+ </div>
+ </footer>
```

Comment:
Indentation! Take a look at our [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e).

```
<footer>
  <div>
    <a href="https://web.whatsapp.com" target="_blank">
      <img src="./images/icon-social/social-icon-whatsapp.jpg" alt="Whatsapp" class="social-icon">
    </a>
    <a href="https://twitter.com" target="_blank">
      <img src="./images/icon-social/social-icon-twitter.jpg" alt="Twitter" class="social-icon">
    </a>
    <a href="https://facebook.com" target="_blank">
      <img src="./images/icon-social/social-icon-fb.jpg" alt="Facebook" class="social-icon">
    </a>
    <a href="https://instagram.com" target="_blank">
      <img src="./images/icon-social/social-icon-insta.jpg" alt="Instagram" class="social-icon">
    </a>
    <a href="https://pinterest.com" target="_blank">
      <img src="./images/icon-social/social-icon-pinterest.jpg" alt="Pinterest" class="social-icon">
    </a>
    <a href="https://youtube.com" target="_blank">
      <img src="./images/icon-social/social-icon-youtube.jpg" alt="Youtube" class="social-icon">
    </a>
    <h2 class="footer-h2">Stressed is just desserts spelled backwards</h2>
  </div>
  <hr>
  <div>
    <p class="p-footer">Page created 10/2020 by Mirjam B. for <a href="https://codeexcursion.ch" target="_blank">Code Excursion</a>
    </p>
 </div>
</footer>
```